documentation/src/mkdocs.yml

```diff
@@ -64,3 +70,5 @@ extra:
     version: 0.1
 
 google_analytics: ['UA-1508605-1', 'pravega.io']
+
+pravega_samples_verion: v0.1.1
```

Comment:
Probably wrong variable name: do you mean `pravega_samples_version`?

Reply:
the variables from `.yml` cannot be rendered in `.md` files. This is a request which is not addressed yet. Ref : https://github.com/mkdocs/mkdocs/issues/304

For now, I have hard coded the version number, in future, we need to be able to customize the version.